services/m3aggregator/config/aggregator.go

```diff
@@ -46,6 +46,10 @@ import (
 	"github.com/spaolacci/murmur3"
 )
 
+const (
+	initialBufferSizeGrowthFactor = 2
```

Comment:
Why 1.5 times? I think it roughly doubles the capacity if you write past the pre-allocated capacity.